client/layout/guided-tours/positioning.js

```diff
@@ -156,8 +156,9 @@ function validatePlacement( placement, target ) {
 		: placement;
 }
 
-function scrollIntoView( target ) {
-	const container = getScrollableSidebar();
+function scrollIntoView( target, scrollContainer ) {
+	// TODO(lsinger): consider replacing with http://yiminghe.me/dom-scroll-into-view/
+	const container = scrollContainer || getScrollableSidebar();
```

Comment:
Not here, that's right. But in `config-elements.js` we have:

```JavaScript
const stepPos = getStepPosition( { placement, targetSlug: target, shouldScrollTo, scrollContainer: this.scrollContainer } );
```

... and this is fine because:

```JavaScript
	componentWillMount() {
		...
		this.scrollContainer = query( this.props.scrollContainer )[ 0 ] || global.window;
		...
	}
```